src/cla/tcp/mod.rs

```diff
@@ -591,15 +605,37 @@ impl Listener {
 }
 
 impl TcpConvergenceLayer {
-    pub fn new(port: Option<u16>, refuse_existing_bundles: bool) -> TcpConvergenceLayer {
+    pub fn new(local_settings: Option<&HashMap<String, String>>) -> TcpConvergenceLayer {
+        let port = local_settings
+            .and_then(|settings| settings.get("port"))
+            .and_then(|port_str| port_str.parse::<u16>().ok())
+            .unwrap_or(4556);
+        let local_refuse_existing_bundles = local_settings
+            .and_then(|settings| settings.get("refuse-existing-bundles"))
+            .and_then(|val| val.parse::<bool>().ok());
+        let global_refuse_existing_bundles = (*CONFIG.lock())
+            .cla_global_settings
+            .get(&super::ConvergenceLayerAgents::TcpConvergenceLayer)
+            .and_then(|settings| settings.get("refuse-existing-bundles"))
+            .and_then(|ref_str| ref_str.parse::<bool>().ok())
+            .unwrap_or(false);
+        let refuse_existing_bundles =
+            local_refuse_existing_bundles.unwrap_or(global_refuse_existing_bundles);
+        debug!(
+            "Extension settings: {:?}",
+            (*CONFIG.lock()).cla_global_settings
+        );
         TcpConvergenceLayer {
-            local_port: port.unwrap_or(4556),
-            sessions: Arc::new(Mutex::new(HashMap::new())),
+            local_port: port,
             listener: Arc::new(Mutex::new(None)),
             refuse_existing_bundles,
         }
     }
 
+    pub fn ext_help_str() -> &'static str {
+        "port=4556:refuse-existing-bundles=false"
```

Comment:
why no help for the local `-C` variant? port should only be local and not globally